cardano-node/src/Cardano/Node/Configuration/TopologyP2P.hs

```diff
@@ -259,6 +276,14 @@ readTopologyFile tr nc = do
     , "configuration flag. "
     , Text.pack err
     ]
+  handlerBootstrap :: Text
+  handlerBootstrap = mconcat
+    [ "You seem to have not configured any trustable peer. "
+    , "This is important in order for the node to make progress "
+    , "in bootstrap mode. Make sure you provide at least one bootstrap peer "
+    , "source (by setting useBootstrapPeer topology file option) "
+    , "or mark a local root peer as trustable. "
+    ]
```

Comment:
> `bootstrap mode`: do we explain somewhere what is boostrap mode? Or maybe it would be better to say:
> 
> > This is important for the node to be able to make progress when syncing.

I have a pr for cardano-node-wiki where I explain what boot strap peers are: https://github.com/input-output-hk/cardano-node-wiki/pull/39/files#diff-743fb54f52af163e2ec1e256e645df88e52976e97c677f5aaa461821c4768f58R157-R189